opm-bundle/manifests/local-storage-operator.clusterserviceversion.yaml

```diff
@@ -305,7 +306,7 @@ spec:
                 serviceAccountName: local-storage-operator
                 containers:
                   - name: local-storage-operator
-                    image: quay.io/openshift/origin-local-storage-operator:latest
+                    image: quay.io/huffmanca/lso:latest
                     ports:
```

Comment:
we should leave this image names alone though..

Reply:
Apologies. These were leftover from testing, but should be resolved.